tests/MongoDB.Driver.Examples/CausalConsistencyExamples.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
```

Comment:
New files require copyright and license.

But... why did this end up in this file? This example has nothing to do with causal consistency.

Reply:
Yep, I will place this test into DocumentationExamples.